README.md

```diff
@@ -1,38 +1,128 @@
 # BentoMap
+> Map Clustering for Swift
 
 [![Build Status](https://travis-ci.org/Raizlabs/BentoMap.svg?branch=develop)](https://travis-ci.org/Raizlabs/BentoMap)
 [![Version](https://img.shields.io/cocoapods/v/BentoMap.svg?style=flat)](http://cocoapods.org/pods/BentoMap)
 [![License](https://img.shields.io/cocoapods/l/BentoMap.svg?style=flat)](http://cocoapods.org/pods/BentoMap)
 [![Platform](https://img.shields.io/cocoapods/p/BentoMap.svg?style=flat)](http://cocoapods.org/pods/BentoMap)
 [![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
 
-BentoMap is an implementation of [quadtrees](https://en.wikipedia.org/wiki/Quadtree) for map annotation clustering and storage written in Swift. The library intends to require minimal code in-app to get clustered annotations on screen, the example target included in BentoMap.xcodeproj contains the bare code required dynamically update the map with clusters whenever the map view's bounds changes.
+BentoMap is an implementation of [quadtrees](https://en.wikipedia.org/wiki/Quadtree) originally implemented for map annotation clustering and storage written in Swift.
```

Comment:
#nth simplify:
> BentoMap is a Swift implementation of [quadtrees](https://en.wikipedia.org/wiki/Quadtree), originally implemented for map annotation clustering and storage.
Also, can it be used for anything _other_ than map annotation clustering and storage? Isn't that kind of its thing?